at_client_mobile/README.md

```diff
@@ -136,6 +136,13 @@ flutter:
     - .env
 ```
 
+- If your app supports windows platform then add `biometric_storage` in app's dependencies
```

Comment:
we are adding biometric_storage in pubspec..this dependency will be fetched for all platforms..right?

Reply:
It throws following exception, on windows platform if does not added under dependencies section.

```
flutter: #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:294:7)
<asynchronous suspension>
#1      MethodChannelBiometricStorage.getStorage (package:biometric_storage/src/biometric_storage.dart:318:22)
<asynchronous suspension>
#2      KeyChainManager.readAtClientData (package:at_client_mobile/src/keychain_manager.dart:35:21)
<asynchronous suspension>
#3      KeyChainManager.disableUsingSharedStorage (package:at_client_mobile/src/keychain_manager.dart:84:18)
<asynchronous suspension>
#4      KeyChainManager.initialSetup (package:at_client_mobile/src/keychain_manager.dart:78:7)
<asynchronous suspension>
#5      OnboardingService.initialSetup (package:at_onboarding_flutter/services/onboarding_service.dart:87:5)
<asynchronous suspension>
#6      _AtOnboardingStartScreenState._init (package:at_onboarding_flutter/screen/at_onboarding_start_screen.dart:37:5)
<asynchronous suspension>

```